### PR TITLE
Generate docs

### PR DESCRIPTION
**Pull request by arditdomi**
**Please check if the PR fulfills these requirements**

> - [ ] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [X] Other... Please describe:
run `npm run docbuild` to update the docs

**What is the current behaviour?** (You can also link to an open issue here)

**What is the new behaviour?**

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: docs/content-services/services/audit.service.md [hunk 1]
```diff
@@ -26,31 +26,31 @@ Manages Audit apps and entries.
 -   **getAuditApp**(auditApplicationId: `string`, opts?: `any`): [`Observable`](http://reactivex.io/documentation/observable.html)`<AuditAppEntry>`<br/>
     Get audit application info.
     -   _auditApplicationId:_ `string`  - The identifier of an audit application.
-    -   _opts:_ `any`  - (Optional)Options.
+    -   _opts:_ `any`  - (Optional) (Optional)Options.
     -   **Returns** [`Observable`](http://reactivex.io/documentation/observable.html)`<AuditAppEntry>` - 
 -   **getAuditApps**(opts?: `any`): [`Observable`](http://reactivex.io/documentation/observable.html)`<AuditAppPaging>`<br/>
     List audit applications.
-    -   _opts:_ `any`  - (Optional)Options.
+    -   _opts:_ `any`  - (Optional) (Optional)Options.
```
Comment (DenysVuika) [inline docs/content-services/services/audit.service.md:33]: seems to be generating dublicates for `(Optional)`

File: docs/core/components/viewer.component.md [hunk 2]
```diff
@@ -382,20 +383,18 @@ By default the viewer's zoom scaling is set to 100%.
...
 
-```
-  "viewer": {
-    "enableDownloadPrompt":  true,
-    "enableDownloadPromptReminder": true,
-    "downloadPromptDelay": 50,
-    "downloadPromptReminderDelay": 30
-  }
-```
+      "viewer": {
```
Comment (DenysVuika) [inline docs/core/components/viewer.component.md:392]: it dropped the code ticks

Comment (DenysVuika) [inline docs/core/components/viewer.component.md:392]: missing code block ticks

File: docs/content-services/components/document-list.component.md [hunk 3]
```diff
@@ -792,23 +793,21 @@ This will give the following output:
...
 
 In order to configure the Document List to automatically download the files, the following environment variables would need to be set up in app.config.json - 
 
-```
-"viewer": {
-    "enableFileAutoDownload": true,
-    "fileAutoDownloadSizeThresholdInMB": 15
-}
-```
+    "viewer": {
```
Comment (DenysVuika) [inline docs/content-services/components/document-list.component.md:802]: missing code example ticks

File: docs/process-services-cloud/services/process-filter-cloud.service.md [hunk 4]
```diff
@@ -52,8 +52,8 @@ Manage Process Filters, which are pre-configured Process Instance queries.
 
     -   _value:_ `any`  - 
     -   _filterProperties:_ `string[]`  - 
-    -   _appName:_ `string`  - (Optional)
-    -   _id:_ `string`  - (Optional)
+    -   _appName:_ `string`  - (Optional) (Optional)
```
Comment (DenysVuika) [inline docs/process-services-cloud/services/process-filter-cloud.service.md:55]: Dupliate optional

File: docs/core/services/identity-user.service.md [hunk 5]
```diff
@@ -19,7 +19,7 @@ Gets OAuth2 personal details and roles for users and performs CRUD operations on
 -   **search**(name: `string`, filters?: [`IdentityUserFilterInterface`](../../../lib/process-services-cloud/src/lib/people/services/identity-user-filter.interface.ts)): [`Observable`](http://reactivex.io/documentation/observable.html)`<`[`IdentityUserModel`](../../../lib/process-services-cloud/src/lib/people/models/identity-user.model.ts)`[]>`<br/>
     Search users based on name input and filters.
     -   _name:_ `string`  - Search query string
-    -   _filters:_ [`IdentityUserFilterInterface`](../../../lib/process-services-cloud/src/lib/people/services/identity-user-filter.interface.ts)  - (Optional) 
+    -   _filters:_ [`IdentityUserFilterInterface`](../../../lib/process-services-cloud/src/lib/people/services/identity-user-filter.interface.ts)  - (Optional) (Optional)
```
Comment (DenysVuika) [inline docs/core/services/identity-user.service.md:22]: dupicate "optional"

**Review by alep85 — APPROVED** [review]

**Review by eromano — APPROVED** [review]

**Pull request merged**